cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitorTest.java

```diff
@@ -88,68 +94,67 @@ public void testStateWithOnlyActiveSnapshotWindow() {
+    // two samples for each partition except T1P1. It has no sample in the first window and one in the second window.
+    CruiseControlUnitTestUtils.populateSampleAggregator(2, 4, aggregator, PE_T0P0, 0, WINDOW_MS, METRIC_DEF);
+    CruiseControlUnitTestUtils.populateSampleAggregator(2, 4, aggregator, PE_T0P1, 0, WINDOW_MS, METRIC_DEF);
+    CruiseControlUnitTestUtils.populateSampleAggregator(2, 4, aggregator, PE_T1P0, 0, WINDOW_MS, METRIC_DEF);
 
     LoadMonitorState state = loadMonitor.state(new OperationProgress());
     assertEquals(0, state.numValidMonitoredPartitions());
     assertEquals(0, state.numValidSnapshotWindows());
     assertEquals(1, state.monitoredSnapshotWindows().size());
-    assertEquals(0.5, state.monitoredSnapshotWindows().get(SNAPSHOT_WINDOW_MS), 0.0);
+    assertEquals(0.5, state.monitoredSnapshotWindows().get(WINDOW_MS), 0.0);
 
     // Back fill for T1P1
-    CruiseControlUnitTestUtils.populateSampleAggregator(1, 1, aggregator, T1P1, 0, SNAPSHOT_WINDOW_MS);
+    CruiseControlUnitTestUtils.populateSampleAggregator(1, 1, aggregator, PE_T1P1, 0, WINDOW_MS, METRIC_DEF);
     state = loadMonitor.state(new OperationProgress());
     assertEquals(0, state.numValidMonitoredPartitions());
     assertEquals(1, state.numValidSnapshotWindows());
     assertEquals(1, state.monitoredSnapshotWindows().size());
-    assertEquals(1.0, state.monitoredSnapshotWindows().get(SNAPSHOT_WINDOW_MS), 0.0);
+    assertEquals(1.0, state.monitoredSnapshotWindows().get(WINDOW_MS), 0.0);
 
   }
 
   @Test
   public void testStateWithInvalidSnapshotWindows() {
     TestContext context = prepareContext();
     LoadMonitor loadMonitor = context.loadmonitor();
-    MetricSampleAggregator aggregator = context.aggregator();
+    KafkaMetricSampleAggregator aggregator = context.aggregator();
 
     // populate the metrics aggregator.
     // two samples for each partition except T1P1
-    CruiseControlUnitTestUtils.populateSampleAggregator(3, 4, aggregator, T0P0, 0, SNAPSHOT_WINDOW_MS);
-    CruiseControlUnitTestUtils.populateSampleAggregator(3, 4, aggregator, T0P1, 0, SNAPSHOT_WINDOW_MS);
-    CruiseControlUnitTestUtils.populateSampleAggregator(3, 4, aggregator, T1P0, 0, SNAPSHOT_WINDOW_MS);
-    CruiseControlUnitTestUtils.populateSampleAggregator(3, 1, aggregator, T1P1, 0, SNAPSHOT_WINDOW_MS);
-    CruiseControlUnitTestUtils.populateSampleAggregator(1, 1, aggregator, T1P1, 0, SNAPSHOT_WINDOW_MS);
+    CruiseControlUnitTestUtils.populateSampleAggregator(3, 4, aggregator, PE_T0P0, 0, WINDOW_MS, METRIC_DEF);
+    CruiseControlUnitTestUtils.populateSampleAggregator(3, 4, aggregator, PE_T0P1, 0, WINDOW_MS, METRIC_DEF);
+    CruiseControlUnitTestUtils.populateSampleAggregator(3, 4, aggregator, PE_T1P0, 0, WINDOW_MS, METRIC_DEF);
+    CruiseControlUnitTestUtils.populateSampleAggregator(1, 2, aggregator, PE_T1P1, 0, WINDOW_MS, METRIC_DEF);
+    CruiseControlUnitTestUtils.populateSampleAggregator(1, 2, aggregator, PE_T1P1, 2, WINDOW_MS, METRIC_DEF);
```

Comment:
Are the changes in this and previous line intentional?

Reply:
Yes, because of the reasons explained above.